gen/decode.go

```diff
@@ -128,7 +128,7 @@ func (d *decodeGen) gBase(b *BaseElem) {
 	switch b.Value {
 	case Bytes:
 		if b.Convert {
-			d.p.printf("\n%s, err = dc.ReadBytes([]byte(%s))", tmp, vname)
+			d.p.printf("\n%s, err = dc.ReadBytes(%s)", tmp, tobaseConvert(b))
```

Comment:
For convert mode, the functions being called return two values `([]byte, error)`, so I think this will throw an error.